Update to resolve current published version (1.6.7) issues

elevatejoe opened the pull request:
I incorporated the Async functions you had updated the dev base with.

I have resolved the issue with events parsing and the tab not loading. I incorporated the AutomateDiagnostics.sh script that Noah Tatum put together and completed the lterrors collection portion of it. The settings in the Manifest.xml and (hopefully) all relevant functions have been updated for three OS options (Windows, Mac, Linux).

Commit:
- Incorporating ASYNC changes from DEV channel
- Fixed event data parsing
- Split Linux and Mac diagnostic to resolve python compatibility issues
- Incorporated the LTErrors output
- CSS changes for readability

elevatejoe committed:
Update README.md

- Added steps for allowing the scripts to be run directly off the Control server
- Updated screenshots with the Details and Agent Logs

Commit:
- VSCode formatting

Commit:
- Remove unnecessary console logging
- Fixed OS identification for Windows OS for the Run Diagnostic command
- Expanded OS support to add Linux and Mac OS X separately
- CSS fixes, adjustments

elevatejoe committed:
Update README.md

Commit:
- Setup git ignore

Commit:
Merge branch 'master' of https://github.com/elevatejoe/CWCAutomateDiagnostics

elevatejoe committed:
Delete .vs directory

elevatejoe committed:
Delete .config directory

elevatejoe committed:
Update README.md

- Updated screenshots to reflect most recent changes

elevatejoe committed:
Update README.md

Commit:
- added agent log collection to the .sh script
- fixed typo in setting description
- updated/added comments to changes in Initializer

Commit:
Merge branch 'master' of https://github.com/elevatejoe/CWCAutomateDiagnostics

Commit:
Formatting changes

Commit:
Updated CSS and HTML formatting
Made use of native classes
Reordered Details list

johnduprey commented:
This looks awesome! I'm testing out the changes in my dev environment. Thanks for the contribution!

JohnDuprey committed:
Minor tweaks

Set LTErrors text color to white
Move locationid to separate span for use with reinstall function
Add credit

JohnDuprey committed:
Update Manifest.xml

Change default entries
